platform/linux-generic/m4/configure.m4

```diff
@@ -29,6 +29,8 @@ m4_include([platform/linux-generic/m4/odp_netmap.m4])
 m4_include([platform/linux-generic/m4/odp_dpdk.m4])
 ODP_SCHEDULER
 
+AS_VAR_APPEND([PLAT_DEP_LIBS], ["${LIBCONFIG_LIBS} ${OPENSSL_LIBS}"])
```

Comment:
Should the rest of linux-generic dependency libraries be also listed here? E.g. pcap, rt, pthread, and atomic.

Reply:
I have not noticed any issues with rt, pthread or atomic.

Though I see you've had some issues there:
```
commit 324a2a2c7c1a9d35323483b679eedfebcb310827
Author: Matias Elo <matias.elo@nokia.com>
Date:   Thu Feb 13 10:39:56 2020 +0200

    build: check if libatomic is needed for __atomic_compare_exchange_n
    
    Double wide __atomic_compare_exchange_n() is required by ipfragreass
    example.
    
    Signed-off-by: Matias Elo <matias.elo@nokia.com>
    Reviewed-by: Petri Savolainen <petri.savolainen@nokia.com>
```

I'd have to re-check with pcap. Perhaps I've missed it.

Reply:
OK, I need a bit of help with pcap. If I'm looking at the current code right - there's no way to specify a cross-compile path for pcap, correct? `AC_ARG_WITH([pcap]` is really used for disabling pcap only, the `ODP_PCAP` just assumes `-lpcap` is available in default linking path.

Am I missing something?

Reply:
You are correct regarding the libpcap path. Now, reading the commit message properly, the libtool bug only affects libs with custom install paths. So in that case I guess you are only missing DPDK_LIBS.

Reply:
Fixed in V5.
Bit related question - is there a way to run all docker builds locally without invoking Travis? I'd prefer to check those kind of commits against the whole build matrix but without necessarily pushing them here and hogging Your resources.

Reply:
I've sometimes run the docker images locally for debugging purposes, but there doesn't seem to be an easy method to run the whole build matrix locally.

When Travis is running in your fork of ODP main repo it's not using ODP project resources. So in case you've already submitted a PR, you can create a new branch and cherry pick the updated PR commits (not pushed yet) there. Pushing this new branch to GitHub will trigger Travis in your repo and doesn't take any resources from the ODP project.

Reply:
It seems like you haven't yet enabled Travis in you ODP fork. Travis can be enabled by following [these](https://github.com/OpenDataPlane/odp/wiki/Using-GitHub-and-Travis-for-ODP-Development) instructions.